docs/mor/modules/ROOT/pages/taylor-green-vortex/index.adoc

```diff
@@ -0,0 +1,202 @@
+= 2D Advection-diffusion with Taylor-Green Vortex
+Hassan Ballout, Luca Berti, Christophe Prud'homme
+:page-tags: case
+:page-illustration: taylor-green-vortex/combined_figure.png
+:description: We simulate the 2D Taylor-Green vortex.
```

Comment:
We simulate the advection and diffusion of a passive scalar with 2D Taylor-Green vortex as advection field